apps/extension/src/ui/domains/Account/AccountAdd/Container.tsx

```diff
@@ -185,6 +185,12 @@ const ConnectAccountMethodButtons = () => {
         networks={["polkadot", "ethereum"]}
         to={`/accounts/add/dcent`}
       />
+      <AccountCreateMethodButton
```

Comment:
Currently this component `AccountAdd/Container` is used directly in both the onboarding and the dashboard. So, the Signet option shows up in both places, however because this PR only implements the `AccountAddSignet` components in the dashboard, the onboarding version is just a dead link. You can decide whether you want the option to connect to Signet to be present in the onboarding, and implement the next screens, or else modifiy this component to detect if it's in the onboarding state and hide the signet option.

Reply:
good catch, wasn't aware it's in the onboarding flow, will look 👀

Reply:
Okay added Signet connect to the onboarding flow and tested!